Cargo.lock

```diff
@@ -30,18 +30,18 @@ dependencies = [
 
 [[package]]
 name = "aho-corasick"
-version = "1.1.1"
+version = "1.1.2"
```

Comment:
Please revert the lockfile changes, no reason to do that in the same PR as the rest of this.

Reply:
was not intended.